backend/src/main/java/gov/cdc/usds/simplereport/config/authorization/OrganizationRole.java

```diff
@@ -1,16 +1,30 @@
 package gov.cdc.usds.simplereport.config.authorization;
 
+import java.util.Collections;
+import java.util.EnumSet;
+import java.util.Set;
+
 public enum OrganizationRole {
-    USER("Users"), 
-    ADMIN("Admins"); // ?
+    ENTRY_ONLY("Test-entry users",
+            EnumSet.of(UserPermission.START_TEST, UserPermission.UPDATE_TEST, UserPermission.SUBMIT_TEST)),
+    USER("Users",
+            EnumSet.of(UserPermission.READ_PATIENT_LIST, UserPermission.READ_RESULT_LIST, UserPermission.EDIT_PATIENT,
+                    UserPermission.START_TEST, UserPermission.UPDATE_TEST, UserPermission.SUBMIT_TEST)),
```

Comment:
When is the USER role used? Seems like ENTRY_ONLY fits more closely with the specs:
- https://github.com/CDCgov/prime-simplereport/issues/642
- https://app.zenhub.com/workspaces/simplereport-5ff8be8854ae8b000e378210/issues/cdcgov/prime-simplereport/570

Reply:
That is the default role that everybody holds￼.